extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {RTC_VENDORS} from './callout-vendors.js';
+import {tryParseJson} from '../../../src/json';
+import {dev, user} from '../../../src/log';
+import {Services} from '../../../src/services';
+import {isArray, isObject} from '../../../src/types';
+import {isSecureUrl} from '../../../src/url';
+
+/** @type {string} */
+const TAG = 'real-time-config';
+
+/** @type {number} */
+export const MAX_RTC_CALLOUTS = 5;
+
+/** @enum {string} */
+const RTC_ERROR_ENUM = {
```

Comment:
You'll want to export this so that networks can reference but I suppose you can wait until your doubleclick impl change

Reply:
need to do it now actually, used in tests